docker/_startup_lib.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+verify_config() {
+  # Make sure settings_local.py exists so the app doesn't crash
+  if [ ! -f onisite/settings_local.py ]; then
+    touch onisite/settings_local.py
+  fi
+  # Make sure we have a default urls.py
+  if [ ! -f onisite/urls.py ]; then
+    cp onisite/urls_example.py onisite/urls.py
+  fi
+
+  # Prepare the ENV dir if necessary
+  if [ ! -d /opt/openoni/ENV ]; then
+    /pip-install.sh
+  fi
+}
+
+verify_vars() {
+  # Die on missing APP_URL; this one could default to localhost, except that the
+  # RAIS container won't have gotten it set, making it incapable of reporting the
+  # IIIF URLs to images
+  if [[ ${APP_URL:-} == "" ]]; then
```

Comment:
We should check to see if we need this still given the setting of it in .env file.